features/fixtures/config/bugsnag/all_disabled.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.bugsnag.android.gradle'
 
 bugsnag {
-    endpoint = "http://localhost:${System.env.MOCK_API_PORT}"
-    releasesEndpoint = "http://localhost:${System.env.MOCK_API_PORT}"
+    endpoint = "http://localhost:9339"
```

Comment:
I've hard coded the mock server port, as while we can technically change it there is no practical way or reason to.